components/error_page/error_message.jsx

```diff
@@ -37,7 +37,7 @@ export default function ErrorMessage({type, message, service}) {
                         <li>
                             <FormattedMessage
                                 id='error.local_storage.help3'
-                                defaultMessage='Use a supported browser (IE 11, Chrome 43+, Firefox 38+, Safari 9, Edge)'
+                                defaultMessage='Use a supported browser (IE 11, Chrome 56+, Firefox 60+, Safari 12+, Edge 42+)'
```

Comment:
Desktop app is going to be bumped to updated electron. It's now on 66 so might as well bump it here. 
```suggestion
                                defaultMessage='Use a supported browser (IE 11, Chrome 66+, Firefox 60+, Safari 12+, Edge 42+)'
```

Reply:
Thanks @crspeller, makes sense. Looks like the desktop app master branch is running an Electron version with Chrome v61 though I think there's a plan to bump the Electron version up before the release. 

But I'll wait for the desktop app release and update this PR afterwards with the Chrome version we're using.